src/lib/CommandHandler.ts

```diff
@@ -68,13 +68,14 @@ export class CommandHandler {
             allowCustomValues: parseBoolean(args.allowCustomValues, false),
             warnOnStderr: parseBoolean(args.warnOnStderr, true),
             multiselect: parseBoolean(args.multiselect, false),
+            stdinResolveVars: parseBoolean(args.multiselect, true),
```

Comment:
@MarcelRobitaille a bug!
```suggestion
            stdinResolveVars: parseBoolean(args.stdinResolveVars, true),
```

what's funny is the extension still works correctly if I set `"stdinResolveVars": true/false` in my tests. I wonder if this code does what it's supposed to do.

I think `resolveArgs` should look like:
```
return {
  ...args,
  useFirstResult: parseBoolean(args.useFirstResult, false),
  ...
}
```

to actually override map values set by `...args` with the result of `parseBoolean` calls, i.e. the order matters here. See an example:
```
> a = {'a': 1, 'b': 2}
{ a: 1, b: 2 }
> b = {...a, 'b': 3}
{ a: 1, b: 3 }
> b = {'b': 3, ...a}
{ b: 2, a: 1 } // 'b': 3 is overwritten with a value from a here.
```

Reply:
Yep, you're completely right. I tried to do things too quickly, and the tests could be better.

Reply:
Since apparently I need a second set of eyes looking at this stuff, what do you think of #150 ?